models.py

```diff
@@ -333,7 +333,18 @@ def bounding_box_query(is_thin=False, yield_per=None, **kwargs):
 
         if is_thin:
             markers = markers.options(load_only("id", "longitude", "latitude"))
-        return markers
+
+        if involved_and_vehicles:
+            markers_ids = []
+            for marker in markers:
+                markers_ids.append(marker.id)
+            markers = db_session.query(Marker).filter(Marker.id.in_(markers_ids))
+            vehicles = db_session.query(Vehicle).filter(Vehicle.accident_id.in_(markers_ids))
+            involved = db_session.query(Involved).filter(Involved.accident_id.in_(markers_ids))
+            all_data = (markers.all(), vehicles.all(), involved.all())
+            return all_data
```

Comment:
No need for the variable, just return the three lists with commas in-between (without parentheses)